frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -102,7 +105,8 @@ class BitBox02 extends Component<Props, State> {
         this.setState({ unlockOnly: false });
       }
       if (status === 'seeded') {
-        this.setState({ appStatus: 'createWallet' });
+        console.log('when does this happen: seeded???????', this.state.createOptions);
```

Comment:
Found it, this happens if the BBApp is restarted during setup. Currently it just starts the create-wallet process again. But I think if the user has choosen 12 words or skip sdcard this info is lost...

The user probably would only restart if there was some kind of issue, or if they just forgot (went to lunch)...

Maybe better to go back to setup choices?

cc @benma WDYT?

Reply:
this status event is fired after the bitbox02 password was defined, app restart not necessary. so you can't really go back to the setup choices here. that would rip the user out of the regular setup flow.

Reply:
ah true 🙈

Reply:
In this case, I suggest we just remove `this.setState({ appStatus: 'create-wallet' });` here for the following reason:

appStatus is already  'create-wallet'

restarting the app after passsword was set can then go back to setup choices, which is better for the reason I mentioned before:

>  Currently it just starts the create-wallet process again. But I think if the user has choosen 12 words or skip sdcard this info is lost

Reply:
oups, I noticed my testing is a bit wrong, just reloading the frontend behaves different than restarting servewallet (obviously). Restarting frontend after password goes to set wallet again. Restarting servewallet goes to setup choices, (which is what I proposed anyway).